### PR TITLE
Main-hero near complete. Revise fine detail.

### PR DESCRIPTION
**Pull request by AndresEstebanGonzalez**
Started hero page. Near complete. Have to go back and revise navbar and font sizes.

### CODE REVIEW

File: styles/_main.scss [hunk 1]
```diff
@@ -0,0 +1,39 @@
+.main {
+  margin: 0;
+  padding: 0;
```
Comment (nathan-hyan) [inline styles/_main.scss:3]: To do a reset, it's best to do

```
* {
margin: 0,
padding: 0,
box-sizing: border-box
}
```

This way you can reset margins and paddings of all items in one pass

File: styles/_navbar.scss [hunk 2]
```diff
@@ -1,6 +1,6 @@
 .navbar {
-  position: fixed;
   width: 100vw;
+  margin-top: 0;
```
Comment (nathan-hyan) [inline styles/_navbar.scss:3]: Is it really necessary?

**Review by nathan-hyan — CHANGES_REQUESTED** [review]
Nice stuff, some comments

File: styles/_navbar.scss (hunk 2 above)
Comment (AndresEstebanGonzalez) [reply styles/_navbar.scss:3]: I forgot to remove that line. Justo el navbar me aparecia separado del limit de la pagina. Lo puse para ver si lo arreglaba.

File: styles/_main.scss (hunk 1 above)
Comment (AndresEstebanGonzalez) [reply styles/_main.scss:3]: Explain, please. What is reset?

Comment (nathan-hyan) [reply styles/_main.scss:3]: Every item comes with some value by default. This set of commands, resets every padding and margin of every item back to 0, so it gets easier for you to position things around the screen consistently. Also, `box-sizing: border-box` helps you with that too.

You can read more about it here: https://developer.mozilla.org/es/docs/Web/CSS/box-sizing

**Review by nathan-hyan — APPROVED** [review]
Good job

**Pull request merged**